internal/pkg/image/driver/imagedriver.go

```diff
@@ -459,5 +438,56 @@ func (d *fuseappsDriver) Stop(target string) error {
 	if err = d.overlayFeature.stop(target, false); err != nil {
 		return err
 	}
+	if err = d.gocryptfsFeature.stop(target, false); err != nil {
+		return err
+	}
 	return nil
 }
+
+func KillProcess(pid int, kill bool, name string) error {
```

Comment:
A better name for this would be `StopFuseProcess`.

Reply:
resolved in this commit https://github.com/apptainer/apptainer/pull/1185/commits/9c7ebbd0175ce54089dd7e259bc6b6ef98584c1a